pitch.md

```diff
@@ -1,44 +1,90 @@
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
-
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Coffee drinks are difficult to make. In the world of the ever-present hipster, it's getting harder and harder to get it just right. In a world where customers actually order a "split-shot, heavy on decaf, 20 oz, soy, extra dry latte, with half pump vanilla & half-pump hazlenut", keeping track of all the modifications to make for a coffee drink is a difficult experience even for a physical barista. On the other hand, automatic coffee machines have a finite amount of settings that a customer can customize. 
+
+Alcoholic beverages present a similar challenge. The barista I was discussing this project idea with told me about his favourite drink: "Old fashioned, with 2 shots of rye whisky, cane syrup instead of brown sugar, extra bitters, and orange zest instead of an orange slice". 
+
+If there was a standard language by which a customer could specify a drink to an arbitrary level of precision, then you could easily program an automatic coffee machine or autobartender to make exactly the drink you want. 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+We need a DSL for this because we want to enable users to specify their drinks to arbitrary levels of precision. There are so many factors that go into a caffeinated or alcoholic beverage that we need a language (rather than, for instance, an enumerated type), because there are just so many possibilities. I imagine that there could be a GUI for end users to interact with this system, but compiling to a DSL built specifically for this purpose seems advantageous. For this project in particular, I plan not to build a GUI (because that seems like more trouble than it's worth), but you could certainly imagine building a GUI on top of the basic language.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I **love** coffee, so I'd be really excited to actually make this kind of thing. Amusingly (and probably not coincidentally), I came up with it while sitting in a coffee shop. An important project consideration for me is building something that I think could ulimately be useful, and I think this language could fill a currently empty niche in the drinks market. A barista that I discussed the idea with suggested making the language apply to alcoholic drinks as well, because they're (in his opinion) easier to make algorithmically.
```

Comment:
+1, I think it would be helpful, to prove usefulness, to think of some drinks that could be made with this language and not most coffee machines.